### PR TITLE
I hope this works

### PR DESCRIPTION
**Pull request by llalonde**

### CODE REVIEW

**Comment by dylan-smith** [comment]
Your front matter has a few issues I think.  It should be surrounded by --- (3 dashes).

Also originalUrl shouldn't have a space in it, and you've double'd up on the authorId label.

File: source/_posts/2016-02-25-Congratulations-Xamarin-and-Microsoft.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - mobile
+excerpt: Not long after wrapping up my presentation on cross platform localization strategies at [Confoo][1], my phone started blowing up. Figuratively, not literally.
+comments: true
+authorId: authorId: lori_lalonde
+original url: http://solola.ca/congrats-xam-ms/
+—
+
+Not long after wrapping up my presentation on cross platform localization strategies at [Confoo][1], my phone started blowing up. Figuratively, not literally. <!—more—> I was bombarded with notifications from all angles: Slack (multiple channels), email (multiple accounts), and&nbsp;BBM… yes I still use BBM, but that’s neither here nor there.
+
+I had done my best to ignore the notifications for awhile, since I was on a mission with my fellow [Western Devs][2] buddy, [James Chambers][3], to find Staples – actually,&nbsp;Bureau en&nbsp;Gros, here in Montreal. We were standing in the middle of the Via Rail concourse at the entrance of Bureau en&nbsp;Gros when I finally decided&nbsp;to read&nbsp;what all the fuss was about. This was the moment&nbsp;I received the news about the [Xamarin acquisition by Microsoft][4].
```
Comment (MisterJames) [inline source/_posts/2016-02-25-Congratulations-Xamarin-and-Microsoft.md]: You can likely do a find and replace on &nbsp; with a space.

**Comment by davidwesst** [comment]
Did this ever get merged @llalonde? If not, I can merge it in...although I will probably update the date to be more recent.

**Comment by kbaley** [comment]
[GitHub subscription](https://trello.com/c/JgUjzZfY/10-github-subscription)

**Pull request closed without merging**